easybuild/easyconfigs/c/COSMO_pompa/patch_COSMO_dom_cray_cpu.patch

```diff
@@ -0,0 +1,75 @@
+diff -Nru cosmo_pompa.orig/cosmo/Options cosmo_pompa/cosmo/Options
+--- cosmo_pompa.orig/cosmo/Options	2018-10-04 10:06:23.000000000 +0200
++++ cosmo_pompa/cosmo/Options	2018-10-06 11:59:45.000000000 +0200
+@@ -11,7 +11,4 @@
+ 
+ -include $(ROOT)/Options.lib.cpu
+ 
+-# bit-repro math
+-CMATH_DIR=/users/charpill/prg/mch/cmath
+-FMATH_DIR=/users/charpill/prg/mch/fmath
+-CC = CC
+\ No newline at end of file
++INC     += -I$(MPICH_DIR)/include
+diff -Nru cosmo_pompa.orig/cosmo/Options.lib.cpu cosmo_pompa/cosmo/Options.lib.cpu
+--- cosmo_pompa.orig/cosmo/Options.lib.cpu	2018-10-04 10:06:23.000000000 +0200
++++ cosmo_pompa/cosmo/Options.lib.cpu	2018-10-06 12:04:34.000000000 +0200
+@@ -2,7 +2,6 @@
+ 
+ # global libraries and includes
+ LIB      = $(LDFLAGS)
+-
+ INC      = -I.
+ INC     += -I$(ROOT)/$(SRCDIR)
+ 
+@@ -19,35 +18,34 @@
+ MPII     = -D__MPICH2
+ 
+ # Grib1 library
+-GRIBDWDL = -L$(INSTALL_DIR)/libgrib1 -lgrib1_$(COMPILER)
++GRIBDWDL = -L${EBROOTLIBGRIB1_CRCLIM} -lgrib1_$(COMPILER)
+ GRIBDWDI =
+ 
+ # Grib-API library
+-GRIBAPI_VERSION = v1.13.1.2
+-GRIBAPIL = -L$(INSTALL_DIR)/libgrib_api/$(GRIBAPI_VERSION)/$(COMPILER)/lib -lgrib_api_f90 -lgrib_api -L$(INSTALL_DIR)/libjasper/lib -ljasper
+-GRIBAPII = -I$(INSTALL_DIR)/libgrib_api/$(GRIBAPI_VERSION)/$(COMPILER)/include
++# GRIBAPI_VERSION = v1.13.1
++GRIBAPIL = -L${EBROOTGRIB_API}/lib -lgrib_api_f90 -lgrib_api -L${EBROOTJASPER}/lib -ljasper
++GRIBAPII = -I${EBROOTGRIB_API}/include
+ 
+ # NetCDF library
+-NETCDFL  =
+-NETCDFI  =
++NETCDFL  = -L${NETCDF_DIR}/lib
++NETCDFI  = -I${NETCDF_DIR}/include
```

Comment:
What is the impact of this addition? Why they didn't have it before?

Reply:
I do not know, why it was not used before. But COSMO fails to link if I don't put the corresponding locations here.